src/src.pro

```diff
@@ -359,6 +359,10 @@ else:linux:!android {
     QT += networkauth
     QT += dbus
 
+    system(c++ -lgo 2>&1 | grep "__go_init_main" > /dev/null) {
```

Comment:
I like it, but @oskirby what do you think about this approach?

Reply:
I think this approach looks good. Maybe move this into golang.pri to keep all of the Go stuff in one place and add a comment to explain why we need this.